src/breadcrumb/breadcrumb.component.ts

```diff
@@ -14,6 +14,7 @@ import { BreadcrumbItem } from "./breadcrumb-item.interface";
 import { BreadcrumbItemComponent } from "./breadcrumb-item.component";
 import { DomSanitizer } from "@angular/platform-browser";
 import { Router } from "@angular/router";
+import { I18n } from "carbon-components-angular/i18n";
```

Comment:
You need to `import { I18nModule } from "carbon-components-angular/i18n";` & add to imports in breadcrumb.module.ts as well.